src/components/DayPickerRangeController.jsx

```diff
@@ -187,16 +187,21 @@ export default class DayPickerRangeController extends React.Component {
     } = nextProps;
     let { visibleDays } = this.state;
 
+    let recomputeBlockedDays = false;
```

Comment:
I'd rather have this be named something like `recomputePropModifiers` since it is recomputing more than just the blocked days.